src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -426,5 +426,35 @@
     "op": "replace",
     "path": "/ResourceTypes/AWS::EC2::CarrierGateway/Properties/Tags/Type",
     "value": "Tag"
+  },
+  {
+    "op": "add",
+    "path": "/ResourceTypes/AWS::ECR::PublicRepository/Properties/RepositoryPolicyText/PrimitiveType",
+    "value": "Json"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::SageMaker::Device/Properties/Tags/Type",
+    "value": "List"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::SageMaker::Device/Properties/Tags/ItemType",
+    "value": "Tag"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::SageMaker::DeviceFleet/Properties/Tags/Type",
+    "value": "List"
+  },
+  {
+    "op": "replace",
+    "path": "/ResourceTypes/AWS::SageMaker::DeviceFleet/Properties/Tags/ItemType",
+    "value": "Tag"
+  },
+  {
+    "op": "add",
+    "path": "/ResourceTypes/AWS::SageMaker::ModelPackageGroup/Properties/ModelPackageGroupPolicy/PrimitiveType",
+    "value": "Json"
   }
```

Comment:
linking these as the fixes to https://github.com/aws-cloudformation/cfn-python-lint/pull/1831#issuecomment-748391662